wgengine/tstun/tun.go

```diff
@@ -87,8 +114,14 @@ func WrapTUN(logf logger.Logf, tdev tun.Device) *TUN {
 		closed:         make(chan struct{}),
 		errors:         make(chan error),
 		outbound:       make(chan []byte),
-		filterFlags:    filter.LogAccepts | filter.LogDrops,
+		// TODO(dmytro): (highly rate-limited) hexdumps should happen on unknown packets.
```

Comment:
Be careful with this todo, the packet may contain private information (e.g. could be a misdirected HTTP packet containing basic auth credentials). We should be very careful about logging packet data without explicit consent (e.g. toggle a scary-sounding debug flag).

Reply:
Yes, that makes total sense. I'm just thinking of a scenario where packets get mislabeled as junk (due to something dumb like a rarely-taken branch injecting packets with incorrect `offset`) and us being unable to tell what happened easily.